### ISSUE TITLE
Output to stdout but not the syslog?

### ISSUE THREAD

**Issue by RulerOf** [Motion-Project/motion#1083]
Hi.  The documentation says that an unconfigured `log_file` directive will output to stdout *and* to syslog.

I'm trying to create a systemd unit and I want all of my logs to go into the journal, so I should leave log_file unconfigured and let systemd handle things, but then I'll have motion writing duplicate messages into my journal and my syslog.

Is there any way to output _only_ to stdout?

**Comment by tosiara** [comment]
No, there is currently no way to log into stdout. Only two options available:
1. a file
2. stderr + syslog

**Comment by RulerOf** [comment]
Thanks. I'm logging to a file right now and have configured logrotate, so that'll suit my needs well enough.

This would be a significant improvement for use cases such as systemd units and docker containers, where everything is basically expected to log to stdout.
